### ISSUE TITLE
0.9.3 is not available on PyPi

### ISSUE THREAD

**Issue by zgyarmati** [pixelb/crudini#65]
See at https://pypi.org/project/crudini/
As a number of downstream packaging uses the PyPi as upstream, it would be beneficial to make available also the new release there

**Comment by jdandrea** [comment]
Update: As of 14 March 2020, still seeing 0.9 in pypi:

```
$ pip3 search crudini
crudini (0.9)  - A utility for manipulating ini files
```

**Comment by patlachance** [comment]
@pixelb any reason why to not push latest crudiny versio on PyPi?

**Comment by pgaxatte** [comment]
Any news on this? @pixelb

**Comment by Meetsch** [comment]
The blocker to push on pypi seems to be solved in #58

Pushing to pypi would therefore solve 2 issues ;-)

**Comment by pixelb** [comment]
Pushed to pypi now. Sorry for the delay
